specification/fluidrelay/resource-manager/Microsoft.FluidRelay/preview/2021-03-12-preview/fluidrelay.json

```diff
@@ -0,0 +1,718 @@
+            "description": "Operation completed synchronously.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.FluidRelay/fluidRelayServers/{name}": {
+      "get": {
+        "operationId": "FluidRelayServers_Get",
+        "tags": [
+          "FluidRelayServers"
+        ],
+        "summary": "Get a Fluid Relay server.",
+        "x-ms-examples": {
+          "Get Fluid Relay server details": {
+            "$ref": "./examples/FluidRelayServers_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/FluidRelayServer"
+            },
+            "description": "Request completed synchronously."
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "Error in operation."
+          }
+        }
+      },
+      "put": {
+        "operationId": "FluidRelayServers_CreateOrUpdate",
+        "tags": [
+          "FluidRelayServers"
+        ],
+        "summary": "Create or Update a Fluid Relay server.",
+        "x-ms-examples": {
+          "Create a Fluid Relay server": {
+            "$ref": "./examples/FluidRelayServers_CreateOrUpdate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resource",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FluidRelayServer"
+            },
+            "description": "The details of the Fluid Relay server resource."
+          }
+        ],
+        "responses": {
```

Comment:
>responses [](http://example.com/codeflow?start=9&length=9)

Add 201 status for the create case.

Reply:
We only have syncronous creation call. Can we just keep 200 here? Compliance level of using 201/202 is recommended but not required. https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc015-put-apis-that-only-return-200